cmd/cost/list.go

```diff
@@ -270,6 +309,10 @@ func (o OUCost) printCostPerAccount(awsClient awsprovider.Client) {
 			Unit:      accountCost.Unit,
 		}
 		if o.options.csv {
+			if o.options.claims {
+				fmt.Printf("%s,%s,%s,%s,%s,%s\n", accountCost.accountClaimNamespace, accountCost.accountClaimName, *o.OU.Id, accountCost.AccountID, accountCost.Cost.StringFixed(2), accountCost.Unit)
```

Comment:
Could encompass this into a ToString method in accountCost?

Reply:
yes we could do this, I'll see if I get to following up on it.